static/css/components/buttonSearch.less

```
@@ -0,0 +1,51 @@
+/**
+ * Search button and text input
+ * [TODO: Link to documentation]
+ */
+
+.search-div {
+  // @grey-fafafa;
+  background-color: @white;
+  border: 1px solid @dark-beige;
+  border-radius: .3em;
+  position: relative;
+  display: inline-flex;
+  align-items: center;
+  padding: .2%;
+  padding-left: .5%;
+  width: 50%;
+}
+
+.search-btn-icon {
```

We prefer to use BEM notation for new CSS, so let's make this like so:

```suggestion
.searchbox__btn-icon {
```

And `.searchbox__input`, and so on.